### ISSUE TITLE
deldersveld/topojson is not available anymore, where to get topojson now?

### ISSUE THREAD

**Issue by shivanshPurple** [zcreativelabs/react-simple-maps#360]
https://github.com/deldersveld/topojson
This Repository/page doesn't exist anymore. Where can I get topojson for individual countries with its states now?

**Comment by abhishekti7** [comment]
https://github.com/zcreativelabs/react-simple-maps/issues/345#issuecomment-1747960168

**Comment by shivanshPurple** [comment]
Found out that highcharts topojson available in their documentation also works perfectly with this library.
For example here is a topo json for Germany
[Json Link](https://code.highcharts.com/mapdata/countries/de/de-all.topo.json)

Edit 1: [Documentation link](https://code.highcharts.com/mapdata/)

**Comment by abhishekti7** [comment]
Hi, thanks for the links.

Have you found any resources to show countries along with their cities in the map?

For example, if I zoom in on India in the world map, all the states should be visible

**Comment by shivanshPurple** [comment]
Yes, this highcharts link includes topojson for countries with their states. I have checked for Germany, Italy, India, works fine. For other countries too, it should also be the same case

Edit 1: I am not sure if its possible to do with all the countries shown at once. Maybe you can write up some custom logic for that.
